spec.html

```diff
@@ -29795,6 +29854,24 @@ <h1>Object.getPrototypeOf ( _O_ )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-object.groupby">
+        <h1>Object.groupBy ( _items_, _callbackfn_ )</h1>
+        <emu-note>
+          <p>_callbackfn_ should be a function that accepts two arguments. `groupBy` calls _callbackfn_ once for each element in _items_, in ascending order, and constructs a new object. Each value returned by _callbackfn_ is coerced to a property key. For each such property key, the result object has a property whose name is that key and whose value is an array containing all the elements for which the _callbackfn_ return value coerced to that key.</p>
```

Comment:
```suggestion
          <p>_callbackfn_ should be a function that accepts two arguments. `groupBy` calls _callbackfn_ once for each element in _items_, in ascending order, and constructs a new object. Each value returned by _callbackfn_ is coerced to a property key. For each such property key, the result object has a property whose key is that key and whose value is an array containing all the elements for which the _callbackfn_ return value coerced to that key.</p>
```

Reply:
(because property *name* implies a String)

Reply:
I don't think "name" implies a string, but I'm okay either way.

Reply:
[6.1.7 The Object Type](https://tc39.es/ecma262/#property-name) says "A <i>property name</i> is a property key that is a String."

Reply:
Changed it to "whose key is that property key" since the double key sounded weird.

Reply:
Yeah, that's probably better. But note that Map.groupBy says "whose key is that key", and can't be un-weirded in the same way.